src/main/java/org/apache/commons/net/ftp/FTPHTTPClient.java

```diff
@@ -194,8 +194,8 @@ private BufferedReader tunnelHandshake(final String host, final int port, final
             final String auth = proxyUserName + ":" + proxyPassword;
             final String header = "Proxy-Authorization: Basic " + Base64.getEncoder().encodeToString(auth.getBytes(charset));
             output.write(header.getBytes(charset));
+            output.write(CRLF);
         }
-        output.write(CRLF);
```

Comment:
This one should be kept, https://datatracker.ietf.org/doc/html/rfc7230#autoid-15
> an empty line indicating the end of the header section